src/gurt/debug.c

```diff
@@ -306,7 +306,7 @@ debug_mask_load(const char *mask_name)
 	struct d_debug_bit	*d;
 	struct d_debug_grp	*g;
 
-	D_STRNDUP(mask_str, mask_name, DBG_ENV_MAX_LEN);
+	mask_str = strndup(mask_name, DBG_ENV_MAX_LEN);
```

Comment:
Possibly a comment about why this line should not be changed?

Reply:
done